src/constants/markets.ts

```diff
@@ -113,3 +113,49 @@ export enum FundingDirection {
 export const PREDICTION_MARKET = {
   TRUMPWIN: 'TRUMPWIN-USD',
 };
+
+// Liquidity Tiers
+export const LIQUIDITY_TIERS = {
```

Comment:
What's the long term plan for these? Where is the source of truth?

Reply:
are we going to localize them?

Reply:
They are localized on render :D but we don't have to worry about label anymore in pml